useCase/useCase.go

```diff
@@ -0,0 +1,40 @@
+package useCase
```

Comment:
Avoid camelCasing for packages and folders, even if there are multiple words everything should be lowercase(I don't like it, but that's how Go likes it)